### DISCUSSION TITLE
Problem in running Mesh and Simulation module

### DISCUSSION THREAD

**Discussion by dikshanshs** [neperfepx/neper#236] [Q&A] [answered]
For command $neper -M diracfunc.tess -rcl 0.5 -order 2 
following error is coming  
![image](https://user-images.githubusercontent.com/85054451/120104957-9380f000-c174-11eb-847b-b89b223b51fd.png)

**Comment by rquey** [comment]
Same comments as for #234.

**Comment by dikshanshs** [comment, marked as answer]
Problem solved by updating Neper and FEPX
